### PR TITLE
Add cuBLAS build workflow and fix error causing lines in CMakeLists

### PR DESCRIPTION
**Pull request by RelatedTitle**
Adds a build workflow for whisper.cpp on Windows with cuBLAS support. Removes error causing lines in CMakeLists that prevented building successfully.

Credit: @giterhug for the fix.

https://github.com/ggerganov/whisper.cpp/issues/840

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -235,6 +235,61 @@ jobs:
...
+      - name: Configure
+        run: >
+          cmake -S . -B ./build -A ${{ matrix.arch }}
+          -DCMAKE_BUILD_TYPE=${{ matrix.build }}
+          -DWHISPER_CUBLAS=1
+      
+      - name: Build
+        run: |
+          cd ./build
+          msbuild ALL_BUILD.vcxproj -t:build -p:configuration=${{ matrix.build }} -p:platform=${{ matrix.arch }}
```
Comment (Green-Sky) [inline .github/workflows/build.yml:281]: prefer the use of `cmake --build`

**Review by Green-Sky — CHANGES_REQUESTED** [review]

**Comment by Green-Sky** [comment]
xref llama.cpp https://github.com/ggerganov/llama.cpp/pull/1271

File: .github/workflows/build.yml (hunk 1 above)
Comment (RelatedTitle) [reply .github/workflows/build.yml:281]: Could you explain why this is better than the current one? I'm not really that well-versed in CMake or compiling C++ projects in general so I just used the same as the other Windows builds.

Comment (Green-Sky) [reply .github/workflows/build.yml:281]: It makes it buildsystem agnostic. In theory we could switch to eg. Ninja, without touching that line.
Consider it best practice.

Comment (RelatedTitle) [reply .github/workflows/build.yml:281]: Hmm, wouldn't it be best to switch all the builds to that all at once?

Comment (ggerganov) [reply .github/workflows/build.yml:281]: Sure, we can do that in a follow up PR

**Review by ggerganov — APPROVED** [review]

**Pull request merged**